overops-functions/build.gradle

```diff
@@ -14,7 +14,9 @@ buildscript {
 
 dependencies {
 	compile (
-		"org.apache.commons:commons-lang3:3.7"
+		"org.apache.commons:commons-lang3:3.7",
+		"com.atlassian.jira:jira-rest-java-client:2.0.0-m2",
+		"com.amazonaws:aws-lambda-java-core:1.2.0"
```

Comment:
Because the dependencies are being bundled into the jar file on creation, please remove the `lambda` one, which is only being used for testing I believe

Reply:
done 👍